src/toil/utils/toilDebugFile.py

```diff
@@ -0,0 +1,144 @@
+# Copyright (C) 2017- Regents of the University of California
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Debug tool for copying files contained in a toil jobStore.
+"""
+
+from __future__ import absolute_import
+import logging
+import fnmatch
+import os.path
+
+from toil.lib.bioio import getBasicOptionParser
+from toil.lib.bioio import parseBasicOptions
+from toil.common import Toil, jobStoreLocatorHelp, Config
+from toil.version import version
+
+logger = logging.getLogger( __name__ )
+
+def recursiveGlob(directoryname, glob_pattern):
```

Comment:
More style stuff that should really be addressed in a separate linting PR or something: PEP8 recommends between global functions. Feel free to ignore this